pangeo_forge_recipes/executors/dask.py

```diff
@@ -43,36 +43,35 @@ def compile(pipeline: Pipeline):
 
         # start with just the config as a standalone layer
         # create a custom delayed object for the config
-        config_task = "config"
-        config_token = append_token(config_task, token)
-        layers[config_task] = {config_token: pipeline.config}
-        dependencies[config_task] = set()
+        config_key = append_token("config", token)
+        layers[config_key] = {config_key: pipeline.config}
+        dependencies[config_key] = set()
 
-        prev_token = ()  # type: Tuple[str, ...]
-        prev_task = config_task
+        prev_key = config_key
+        prev_dependency = ()  # type: Union[Tuple[()], Tuple[str]]
```

Comment:
Just for my own curiosity, just having `Tuple[str, ...]` didn't work? I would have thought that a length-zero tuple would be allowed.

Reply:
Although, I guess this is slightly better, since it's explicitly either a size-0 empty tuple or a size-1 tuple (with a str).

Reply:
> Just for my own curiosity, just having `Tuple[str, ...]` didn't work?

Never tried it. For me mypy compliance is usually a matter of trial and error. 😬